File: week6/pset6/sentimental-mario-more/mario.py
# TODO: mario-more

from cs50 import get_int

# prompt user for input
h = get_int("Height: ")

# check input for condition if not prompt again
while (h < 1 or h > 8):
    h = get_int("Height: ")

# print shape
for i in range(h):
    for j in range(h, i + 1, -1):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    for l in range(2):
        print(" ", end="")
    for m in range(i + 1):
        print("#", end="")
    print()

File: week1/pset1/mario-less/mario.c
// Problem Set 1
#include <cs50.h>
#include <stdio.h>

int main(void)
{
// Prompt user for Height
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

// Printing Block
    for (int i = 0; i < n; i++)
    {
        for (int j = n; j > i + 1; j--)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}